sdk/appcomplianceautomation/azure-resourcemanager-appcomplianceautomation/src/main/java/module-info.java

```diff
@@ -10,7 +10,6 @@
     exports com.azure.resourcemanager.appcomplianceautomation.fluent.models;
     exports com.azure.resourcemanager.appcomplianceautomation.models;
 
-    opens com.azure.resourcemanager.appcomplianceautomation.fluent.models
-        to com.azure.core, com.fasterxml.jackson.databind;
+    opens com.azure.resourcemanager.appcomplianceautomation.fluent.models to com.azure.core;
```

Comment:
- change that got omitted by https://github.com/Azure/azure-sdk-for-java/pull/43500